### PR TITLE
Setup API

### PR DESCRIPTION
**Pull request by addisalemseifu**
-Register data to the `API` with the `POST` method.
-Fetched data from `API` with the `GET` method.
apshot

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -11,32 +11,40 @@
...
-                    <input class="submit" type="submit">
-                </div>
-            </div>
+                <div class="error-holder"></div>
+            <!-- <h1> Leader Board</h1> -->
+            
     </header>
+
+  <script src="./index.js"></script>
+  <!-- <script src="./indexAxios.js"></script> -->
```
Comment (ichala) [inline src/index.html]: - [ ] To keep your repo clean kindly remove any commented code from your project 🙏

File: src/index.js [hunk 2]
```diff
@@ -1 +1,56 @@
...
+        userdatails.innerHTML += `
+      <div class="my-div">${el.user}:${el.score}</div>`;
+      });
+      const myDiv = document.getElementsByClassName('my-div');
+      for (let i = 0; i < myDiv.length; i += 1) {
+        if (i % 2 === 0) {
+          myDiv[i].classList.add('active');
+        }
+      }
+    });
```
Comment (ichala) [inline src/index.js]: - [ ] As part of the requirements you need to use async and await with the fetch method to send and receive data , kindly fix it 🙏

**Review by ichala — CHANGES_REQUESTED** [review]
Hi Student
Good job so far! 
👏👏👏 
You still need to work on some issues to go to the next project, but you're almost there!

the-simpsons-homer-simpson

## To Highlight 🎉

- No Linter errors  ✔️ 
- Send & Recieve Works ✔️


 ## Required Changes ♻️


- [ ] **_🛑  Check The Comments Below_**





## Optional suggestions

- N/A

**********
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by OyePriscilla — APPROVED** [review]
Hi @addisalemseifu 👏 👏, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
Send and receive data from an API ✔️ 
Use `async` and `await` with the `fetch` method to send and receive data ✔️ 
App runs as expected ✔️ 

## Optional suggestions

- [optional] Kindly add the missing Key features header with relevant content to your README file, this will give us an overview of what's your whole project will achieve. 👍 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**